app/views/notes/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% notes.each_with_index do |note, i| %>
+  <h3>Note <%= i+1 %>:</h3>
+  <div>Name: <%= "#{note.name}.#{note.file_type}" %></div>
+  <div>Author: <%= note.user_id %></div>
+  <div>Category: <%= note.category_id %></div>
```

Comment:
This should be note.category.name. I know this doesn't exist yet but I don't think we need to display the id of a category.